src/vstest.console/Processors/DisableAutoFakesArgumentProcessor.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors
+{
+    using System;
+    using System.Globalization;
+
+    using CommandLineResources = Microsoft.VisualStudio.TestPlatform.CommandLine.Resources.Resources;
+
+    /// <summary>
+    /// An argument processor that allows the user to disable fakes
+    /// from the command line using the --DisableAutoFakes|/DisableAutoFakes command line switch.
+    /// </summary>
+    internal class DisableAutoFakesArgumentProcessor : IArgumentProcessor
```

Comment:
I think this will break cloud build if we don't auto enable fakes. Is it a possibility that this can be done in the caller of `vstest.console.exe`?